File: examples/dotnet-jit/StdinEcho.cs
// Read from stdin and echo each line.
var lines = new List<string>();
string? line;
while ((line = Console.ReadLine()) != null)
{
    lines.Add(line);
}
Console.WriteLine($"lines={lines.Count}");
foreach (var l in lines)
{
    Console.WriteLine($"echo: {l}");
}
Console.WriteLine("stdin-done");

File: examples/node/stdin_echo.js
/** Read from stdin and echo each line. */
const readline = require('readline');
const rl = readline.createInterface({ input: process.stdin });
const lines = [];
rl.on('line', (line) => { lines.push(line); });
rl.on('close', () => {
    console.log(`lines=${lines.length}`);
    for (const line of lines) { console.log(`echo: ${line}`); }
    console.log('stdin-done');
});
